index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <main class="container">
+        <header class="header-background">
+            <div class="logo-placeholder">
+                <a href="/index.html">Mark</a>
+            </div>
+            <nav class="icon-menu">
+                    <div class="menu-icon">
+                        <img src="./images/menu.svg" alt="menu icon" class="nav-icon">
+                    </div>
+            </nav>
+        </header>
+        <section class="headline">
+            <div class="name">
+                <h1 class="username">I'm Mark, Glad to see you!</h1>
+            </div>
+            <div class="description">
+                <p class="user-description">
+                    I’m a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+            </div>
+            <span class="socialmedia">LET'S CONNECT</span>
+            <div class="social-media-icons">
+                <a href="" class="twitter-link">
+                    <img src="./images/twitter.svg" alt="twitter" class="twitter" width="20px" height="20px" >
+                </a>
+                <a href="" class="linkedin-link">
+                    <img src="./images/linkedin.svg" alt="linkedin" class="linkedin" width="20px" height="20px">
+                </a>
+                <a href="" class="email-link">
+                    <img src="./images/email.svg" alt="email" class="email" width="20px" height="20px">
+                </a>
+                <a href="" class="social-media-link">
+                    <img src="./images/github.svg" alt="github" class="github" width="20px" height="20px" >
+                </a> 
+            </div>
```

Comment:
[OPTIONAL]
- Please remember to place these icons in an `ul` tag with each icon having a `li` tag.